src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.cpp

```diff
@@ -11,6 +11,8 @@
 
 #include "src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.hpp"
 
+std::map<ModuleOp, std::unique_ptr<FunctionToInitStates>> initMap;
```

Comment:
Can you comment on this map, when init, what does it hold, what is the duration of when the map is valid. Tx